install.js

```diff
@@ -228,6 +227,21 @@ function requestBinary(requestOptions, filePath) {
 }
 
 
+function get(requestOptions, callback, redirects) {
+  redirects = redirects || 0;
+  var protocol = requestOptions.protocol === 'https:' ? https : http;
+  protocol.get(requestOptions, function (response) {
+    var status = response.statusCode;
+    if ((status === 302 || status === 301 || status === 307) && redirects < 5) {
+      console.log('Redirect to %s', response.headers.location);
+      redirects++;
+      return get(url.parse(response.headers.location), callback, redirects);
```

Comment:
You can't use `url.parse(response.headers.location)` here as this will ignore all other request options. You need to use `getRequestOptions`.